internal/handlers/pg/msg_delete.go

```diff
@@ -145,11 +149,26 @@ func (h *Handler) MsgDelete(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 			return nil
 		})
 
+		return err
```

Comment:
I believe, if we keep this `processQuery` func here, instead of this return `err` we can just use the `return` statement on the line 103 (where `inTransaction` is called).

Reply:
Correct! Thank you